procurement_analytic/README.rst

```diff
@@ -0,0 +1,63 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+====================
+Procurement Analytic
+====================
+
+This module adds account analytic field in procurements. This field can be
+used for propagating this data to the generated objects, like purchase orders
+or manufacturing orders.
+
+
+Configuration
+=============
+
+To configure this module, you need to:
+
+#. Go to your user settings.
+#. Enable *Analytic Accounting for Purchases* in *Technical Settings'.
```

Comment:
I need to remove Configuration and Usage sections because this module per se, without install inventory, purchase or sale modules, It can't be configured.